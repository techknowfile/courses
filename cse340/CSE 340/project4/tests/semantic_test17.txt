VAR
    a, b, c : INT;
    d : BOOLEAN;
    e, f : REAL;
    g, h : LONG;
{
    a = b;
}

