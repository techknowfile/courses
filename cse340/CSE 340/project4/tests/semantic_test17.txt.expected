BOOLEAN d #
INT a b c #
LONG g h #
REAL e f #
STRING #
